### DISCUSSION TITLE
Oraxen Ore generation help

### DISCUSSION THREAD

**Discussion by itsMarkiS** [DerFrZocker/Custom-Ore-Generator#13] [Q&A]
is there any example of how to use oraxen ores????
i was following the docs but it is kinda hard to understand...

and when i finally used the command to apply the ore config in my world, actually nothing happened, the ores were not generating and stuff like that....

also theres almost no information on how to make it work, neither a tutorial for it....

**Comment by DerFrZocker** [comment]
To generated a Oraxen block follow the following steps:

1. Place the custom block
2. Use `/oregen create`
3. Enter a name like `MyCustomOre`
4. Left click the custom block you placed
5. Type `!next`
6. Left click the blocks you want to replace such as `stone` and `deepslate`
7. Type `!next`
8. Type `!next` again
For normal world generation:
9.  Select `Vanilla Minable Generator` (diamond block)
10. Select `Count Range` (iron block)
11. When it should generated in all biomes click `next` (stick)
12. Set a `vein size` (coal) and click `next` (stick)
13. Set a `height range`, `veins per chunk` and `minimum height` (coal) and click `next` (stick)
14. Click on `Oraxen`
15. Click on `Click to set the found Custom Data` (enchantet bock)
16. Click `next` (stick)
17. Click the world in which the ore should generated
18. Click `next` (stick)
19. Click `Click to create the ore-config` (upper green block)

If your Oraxen block does not generated in new chunks, then there might be an error with you configuration. In this case please send the configuration file found in the directory: `plugins\custom-ore-generator\data\ore-config`
